recipe/recipe.yaml

```diff
@@ -0,0 +1,79 @@
+context:
+  name: qdldl-python
+  version: 0.1.7.post5
+  lib_ver: ${{ (version | split("."))[:3] | join(".") }}
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/osqp/${{ name }}/archive/refs/tags/v${{ version }}.tar.gz
```

Comment:
Please replace all instances of `${{ name }}` with it's actual value and remove the variable.